snapshot/snapshot.go

```diff
@@ -82,6 +84,19 @@ func NewSnapshotter(ctx context.Context, cfg *config.SnapshotterConfig) (snapsho
 		return nil, errors.Wrap(err, "parse recover policy")
 	}
 
+	cgroupConfig, err := config.ParseCgroupConfig(cfg.CgroupConfig)
+	if err != nil {
+		return nil, errors.Wrap(err, "parse cgroup configuration")
+	}
+	log.L.Infof("parsed cgroup config: %#v", cgroupConfig)
+	cgroupMgr, err := cgroup.NewManager(cgroup.Opt{
+		Name:   "nydusd",
+		Config: cgroupConfig,
+	})
+	if err != nil && (err != cgroup.ErrCgroupNotSupported || err != v2.ErrRootMemorySubtreeControllerNotEnableed) {
```

Comment:
typo “Enableed”  or this can be named to xxxDisabled

Reply:
Good point. Fixed.